packages/frontend/model/article-sections.test.ts

```diff
@@ -0,0 +1,159 @@
+import { findBySubsection } from '@frontend/model/article-sections';
+
+describe('returns section for each subsection', () => {
```

Comment:
What about using a table driven test - e.g. see like https://dave.cheney.net/2013/06/09/writing-table-driven-tests-in-go.

    const testCases = [
        [['books', 'childrens-books-site'], 'Books'],
        ...
    ];

    // test and asset by mapping over here

That way things become a lot more concise and it is easier to add/remove cases. The disadvantage is that error reporting on failing cases is slightly less clear, but it should still be pretty obvious what is going wrong.

Reply:
Yes agree is much neater using a table. Refactored it